it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/dataflow/DirectRunnerClient.java

```diff
@@ -109,6 +110,12 @@ public JobState getJobStatus(String project, String region, String jobId) {
     return managedJobs.get(jobId).getJobState();
   }
 
+  @Override
+  public List<JobMessage> listMessages(
+      String project, String region, String jobId, String minimumImportance) {
+    return null;
```

Comment:
consider return an empty list to avoid possible NPE (and comply with nullness check)?

Reply:
Done